src/main/kotlin/no/nav/familie/ef/sak/api/oppgave/OppgaveController.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.api.oppgave
+
+import no.nav.familie.ef.sak.service.OppgaveService
+import no.nav.familie.ef.sak.util.RessursUtils.ok
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.kontrakter.felles.oppgave.FinnOppgaveResponseDto
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/oppgave")
+@ProtectedWithClaims(issuer = "azuread")
+@Validated
+class OppgaveController(val oppgaveService: OppgaveService) {
+
+    @PostMapping(path = ["/hent-oppgaver"],
+                 consumes = [MediaType.APPLICATION_JSON_VALUE],
+                 produces = [MediaType.APPLICATION_JSON_VALUE])
+    fun hentOppgaver(@RequestBody finnOppgaveRequest: FinnOppgaveRequestDto): ResponseEntity<Ressurs<FinnOppgaveResponseDto>> {
```

Comment:
Fjern ResponseEntiy<> i return statement. Blir bare støy i koden. Samme gjelder de to metodene under

Reply:
Vet ikke om jeg synes det er støy - ResponseEntity er vel brukt på alle applikasjonene på tvers av teamene? Eller misforstår jeg noe?

Reply:
Jeg har prøvdd å fjerne det alle steder jeg har kommet over det. Returobjekter wrappes i responseEntity av spring uansett. Ved å definere det i returtypen så må du også legge det til manuelt.

Reply:
Ok, fjernet nå :)